### ISSUE TITLE
Crash when using too many images

### ISSUE THREAD

**Issue by GPMueller** [spirit-code/spirit#187]
It appears that too many images are not handled well by the code - need to check if this has to do with too little memory space left.
```
2017-04-13 21:26:00  [ DEBUG ] [API ] [01] [11]  Switched to next image 11 of 11
2017-04-13 21:26:03  [ INFO  ] [API ] [01] [11]  Copied image to clipboard.
Segmentation fault: 11
```
Another error which appears:
```
2017-04-13 20:42:43  [ INFO  ] [API ] [01] [--]  Deleted image 11 of 11
libc++abi.dylib: terminating
Abort trap: 6
```

"hm, also, was ich mache ist folgendes: 
1. spirit starten mit 1 image 
2. spinchain mit 11 images laden
3. versuchen, ein 12. image zu erstellen
4. crash"

**Comment by GPMueller** [comment]
This seems to primarily appear when using a chain which was read from file.

**Comment by GPMueller** [comment]
Hopefully fixed by d5d08614b7ff1cddef6ea5cd2f9a30c8985bb1f6.
